projects/README.md

```diff
@@ -6,6 +6,19 @@ This directory acts as an overview of the projects that the Alliance hosts.
 
 > **Note:** This directory is currently a simple list of links to websites or repositories. We are working on a more sophisticated way of presenting this information.
 
+## Core Projects
+
+ * [Wasmtime](https://wasmtime.dev/)
+   * Repositories:
+     * [wasmtime](https://github.com/bytecodealliance/wasmtime)
+     * [wasmtime-cpp](https://github.com/bytecodealliance/wasmtime-cpp)
```

Comment:
```suggestion
```

This line should go away once https://github.com/bytecodealliance/wasmtime/pull/10582 is merged.

Reply:
Let's remove it now, then: no need to wait until the deed is done in this case :)